### DISCUSSION TITLE
Adding hydrogen to charge species

### DISCUSSION THREAD

**Discussion by pratik994** [rdkit/rdkit#4976] [Q&A]
Hi all, 

Is there a way to add hydrogen to the nitrogen atom as well while keeping the charge to -1 ?

![Screenshot 2022-02-01 131955](https://user-images.githubusercontent.com/46387157/152036192-0379609a-68b5-4e59-8b45-57e7bf60b073.png)

**Comment by gedeck** [comment]
Did you mean?
```
mol = Chem.MolFromSmiles('CCC(C)O[NH-]')
Chem.AddHs(mol, explicitOnly=False)
```
![image](https://user-images.githubusercontent.com/8720575/152040551-5c673979-caf0-4f60-ae02-feaff07c4386.png)

`[N-]` defines a nitrogen without hydrogens, a radical cation.

**Reply by pratik994** [reply]
Thanks for the prompt reply.

I guess what I meant was is there a way to add hydrogen atom to the [N-] radical?
